portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,100 @@
   width: 650px;
 }
 
+h1 {
+	font-size: 50px;
```

Comment:
The inconsistent indentation here is a bit jarring. Looks like it's because there's a tab instead of spaces. I generally prefer spaces to tabs, but again, consistency is key.